### PR TITLE
Aboutlibraries

### PR DESCRIPTION
**Pull request by timc-mozilla**
Open Source Libraries

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -18,11 +18,13 @@ kotlin = "1.9.20"
 kotlin-ksp = "1.9.20-1.0.14" # Used only once, but keeping this definition makes the KSP version check at ci/verify-ksp-version.sh simpler.
 kotlinx-coroutines = "1.7.3"
 okhttp = "4.11.0"
+oss-licenses-plugin = "0.10.6"
```
Comment (JohnOberhauser) [inline gradle/libs.versions.toml:21]: Can you remove all the oss stuff if we are not using it?

File: settings.gradle.kts [hunk 2]
```diff
@@ -14,6 +14,9 @@ pluginManagement {
             if (requested.id.id == "org.mozilla.telemetry.glean-gradle-plugin") {
                 useModule("org.mozilla.telemetry:glean-gradle-plugin:${requested.version}")
             }
+            if (requested.id.id == "com.google.android.gms.oss-licenses-plugin") {
```
Comment (JohnOberhauser) [inline settings.gradle.kts:17]: Can you remove this?

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -13,7 +13,8 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/Theme.MozillaSocial"
-        tools:targetApi="31">
+        android:enableOnBackInvokedCallback="true"
```
Comment (JohnOberhauser) [inline app/src/main/AndroidManifest.xml:16]: What does this do?

Comment (timc-mozilla) [reply app/src/main/AndroidManifest.xml:16]: residual from testing different things for AboutLibraries. will remove.

**Comment by JohnOberhauser** [comment]
One more thing is that there is no dark mode.  You can pass our themed colors into the `LibrariesContainer`

**Review by JohnOberhauser — APPROVED** [review]

**Review by JohnOberhauser — APPROVED** [review]

**Pull request merged**